packages/extension-base/src/background/handlers/State.ts

```diff
@@ -368,19 +367,20 @@ export default class State {
     const idStr = this.stripUrl(url);
 
     // Do not enqueue duplicate authorization requests.
-    const isDuplicate = Object.values(this.#authRequests)
+    const isDuplicate = Object
+      .values(this.#authRequests)
       .some((request) => request.idStr === idStr);
 
     assert(!isDuplicate, `The source ${url} has a pending authorization request`);
 
     if (this.#authUrls[idStr]) {
       // this url was seen in the past
-      assert(this.#authUrls[idStr].authorizedAccounts, `The source ${url} is not allowed to interact with this extension`);
```

Comment:
good point, I must have been missing this in my tests, I expected this part to work for the migration since any old website would have no `authorizedAccounts`, a check for `=== undefined` may have worked as well. But your solution is even more elegant anyway.